### ISSUE TITLE
[Adaptive Cards-InputForm.json]: Error suggestion message is not provided for the 'Your email' required edit field on after entering any invalid characters on the field present on the 'InputForm.json' card.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9223]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0

**Repro steps:**
1. Open the Adaptive Cards Visualizer application.
2. Navigate to the 'InputForm.json' control present on left navigation using tab key and activate it using enter key.
3. 'InputForm.json' adaptive card will appear on right sight of the pane.
4. Navigate to the 'Your email' required edit field using tab key and enter any invalid characters on the field.
5. Verify that, error suggestion message is provided for the 'Your email' required edit field on after entering any invalid characters on the field present on the 'InputForm.json' card or not.

**Actual Result:**
Error suggestion message is not provided for the 'Your email' required edit field on after entering any invalid characters on the field present on the 'InputForm.json' card. Currently only "Please enter a valid email address" error message alone appears, but no additional suggestion or guidance message is provided for the edit field.

**Note:**
Also "name@example.com" suggestion should be placed as a placeholder for the 'Your email' required edit field.

**Expected Result:**
An appropriate error suggestion message should be provided for the 'Your email' required edit field when invalid characters are entered. In addition to the error message "Please enter a valid email address," a helpful suggestion (e.g., "Use a format like name@example.com") should be displayed adjacent to the error message or the format can be present adjacent to the 'Your email' label text. This ensures better guidance and accessibility for all users, including those using assistive technologies.

**User Impact:**
Users, especially those relying on assistive technologies such as screen readers, may struggle to understand how to correct the input without a clear suggestion message. The lack of guidance can lead to confusion, form submission errors, and a poor user experience. This issue may also prevent users with cognitive disabilities from completing the form efficiently. Overall, it reduces the accessibility and usability of the form for a wide range of users.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/error-suggestion.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/c83ddd99-f79b-4a31-bf4f-84e366c08052

### Card JSON

```JSON
N/A
```
